unreleased_history/public_api_changes/expose_writebatch_iterator_cf_methods_via_c_api.md

```diff
@@ -0,0 +1 @@
+Exposed WriteBatch Iterator column family specific methods: put_cf, delete_cf and merge_cf via C API. Introduced new function `rocksdb_writebatch_iterate_cf` for backwards compatibility.
```

Comment:
Might be a little too much detail. How about "Introduced new C API function `rocksdb_writebatch_iterate_cf` for column family-aware iteration over the contents of a `WriteBatch`"?